app/views/exploreWrapper.scala.html

```diff
@@ -0,0 +1,11 @@
+@import models.user.User
+@(title: String, user: Option[User] = None)(implicit lang: Lang)
+
+@main(title, Some("/explore")) {
+    @navbar(user, Some("/explore"))
+    <link rel="stylesheet" href='@routes.Assets.at("stylesheets/wrapperFrame.css")'/>
+    
+    <iframe id="wrapperFrame"></iframe>
+    
+    <script type="text/javascript" src='@routes.Assets.at("javascripts/common/wrapperFrame.js")'></script> 
+}
```

Comment:
Just make sure to go through and add newlines to the ends of all the files that are missing them!